package.json

```diff
@@ -10,18 +10,15 @@
     "url": "https://github.com/jest-community/jest-watch-typeahead.git"
   },
   "homepage": "https://github.com/jest-community/jest-watch-typeahead",
-  "files": [
-    "build/",
-    "filename.js",
-    "testname.js"
-  ],
+  "files": ["build/", "filename.js", "testname.js"],
   "scripts": {
     "test": "jest",
     "lint": "eslint .",
     "watch": "babel src -w --ignore **/*.test.js,integration -d build",
     "build": "babel src --ignore **/*.test.js,integration -d build",
     "prepublish": "yarn build",
-    "format": "prettier --single-quote --trailing-comma all --write \"!(build)/**/*.js\""
+    "format":
+      "prettier --single-quote --trailing-comma all --write \"!(build)/**/*.js\""
```

Comment:
updating prettier to 1.13 will stop if from formatting package.json differently from yarn/npm

Reply:
Awesome!

Reply:
https://github.com/jest-community/jest-watch-typeahead/pull/17